Linux/generate-csr.sh

```diff
@@ -57,7 +56,10 @@ while getopts ":hjp" opt; do
     case "$opt" in
         h)
 			echo "$(basename "$0") Generate CSR"
-			echo ""
+            echo ""
+			echo "Script Version        : 0.1.0"
+	        echo "Tributech Node Version: 4.0.x"
```

Comment:
Do we really want to outline the tributech version here, seems like its gonna be outdated

Reply:
Lets switch to 4.X + or  > 4.X to indicate that its working from the Tributech Node 4.X upwards